tests/suites/test_suite_psa_crypto_driver_wrappers.function

```diff
@@ -3063,54 +3050,64 @@ void pake_operations(data_t *pw_data, int forced_status_setup_arg, int forced_st
             /* --- psa_pake_input (driver: setup, input) --- */
             mbedtls_test_driver_pake_hooks.forced_setup_status = forced_status_setup;
             mbedtls_test_driver_pake_hooks.forced_status = forced_status;
-            mbedtls_test_driver_pake_hooks.hits = 0;
+            memset(&mbedtls_test_driver_pake_hooks.hits, 0, sizeof(mbedtls_test_driver_pake_hooks.hits));
             TEST_EQUAL(psa_pake_input(&operation, PSA_PAKE_STEP_KEY_SHARE,
                                       input_buffer, size_key_share),
-                       expected_status_setup);
-            TEST_EQUAL(mbedtls_test_driver_pake_hooks.hits, 1);
+                       expected_status);
+            TEST_EQUAL(mbedtls_test_driver_pake_hooks.hits.total, 1);
+            TEST_EQUAL(mbedtls_test_driver_pake_hooks.hits.setup, 1);
             break;
 
         case 1: /* setup (via output) */
-            /* --- psa_pake_input (driver: setup, input) --- */
+            /* --- psa_pake_output (driver: setup, output) --- */
             mbedtls_test_driver_pake_hooks.forced_setup_status = forced_status_setup;
             mbedtls_test_driver_pake_hooks.forced_status = forced_status;
-            mbedtls_test_driver_pake_hooks.hits = 0;
-            TEST_EQUAL(psa_pake_input(&operation, PSA_PAKE_STEP_KEY_SHARE,
-                                      input_buffer, size_key_share),
-                       expected_status_setup);
-            TEST_EQUAL(mbedtls_test_driver_pake_hooks.hits, 1);
+            memset(&mbedtls_test_driver_pake_hooks.hits, 0, sizeof(mbedtls_test_driver_pake_hooks.hits));
+            TEST_EQUAL(psa_pake_output(&operation, PSA_PAKE_STEP_KEY_SHARE,
+                                       output_buffer, output_size, &output_len),
+                       expected_status);
+            TEST_EQUAL(mbedtls_test_driver_pake_hooks.hits.total, 1);
+            TEST_EQUAL(mbedtls_test_driver_pake_hooks.hits.setup, 1);
             break;
 
         case 2: /* input */
```

Comment:
I think it is cleaner if we have separate cases for `setup`, `input`, `output` especially that setup is done indirectly via `input` or `output`. If we optimize this it will be more complicated.